pages/dkp/konvoy/2.2/choose-infrastructure/aks/aks-advanced/aa-prerequisites/index.md

```diff
@@ -12,6 +12,7 @@ enterprise: true
 Before you begin using Konvoy, you must have:
 
 - An x86_64-based Linux or macOS machine.
+- A [Self-managed Azure cluster](../../../azure/advanced/self-managed/)
```

Comment:
There is another reference below (on line 24) that refers to the bootstrap cluster page this PR is deleting.

Reply:
I was wondering about that... i wondered if that was just updating credentials so it should stay in?  So you are saying go ahead and delete the update credentials command `dkp update bootstrap credentials aws`